deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml

```diff
@@ -206,49 +369,25 @@ spec:
               description: 'Number of replicas that the service will have deployed
                 in the cluster Default value: 1'
               format: int32
-              maximum: 100
               minimum: 0
               type: integer
             resources:
-              description: 'The resources for the deployed pods, like memory and cpu
-                Default value: nil'
+              description: Defined Resources for the Jobs Service
```

Comment:
` for the Jobs Service`
Copy/paste ? :)

Reply:
Changed it to [generic description](https://github.com/kiegroup/kogito-cloud-operator/blob/6751a35bf6799b50747e9e5566c06cfe2ea8943f/pkg/apis/app/v1alpha1/kogitoservices_types.go#L128).